torch/csrc/jit/passes/quantization.cpp

```diff
@@ -961,7 +966,8 @@ void FoldPrepackedWeightIntoModule(
     script::Module& module,
     const std::string& method_name,
     const script::Module& linear_params_module,
-    const script::Module& conv_params_module) {
+    const script::Module& conv_params_module,
+    int& uid) {
```

Comment:
Can we just start from 0 in each module? That would avoid the extra argument here (note that to understand what this argument means one needs to know very deep details of the implementation of this function - that's why I'd prefer to avoid adding it).